from django.test import TestCase

# Create your tests here.

""" Use to test that the functions throw the expected Exceptions.
def test_afunction_throws_exception(self):
    self.assertRaises(ExpectedException, afunction, arg1, arg2)

"""